allennlp/tests/models/encoder_decoders/simple_seq2seq_test.py

```diff
@@ -38,9 +38,21 @@ def test_decode_runs_correctly(self):
         # ``decode`` should have added a ``predicted_tokens`` field to ``output_dict``. Checking if it's there.
         assert "predicted_tokens" in decode_output_dict
 
+class SimpleSeq2SeqWithoutAttentionTest_Bidirectional(ModelTestCase):
+    def setUp(self):
+        super().setUp()
+        self.set_up_model(self.FIXTURES_ROOT / "encoder_decoder" / "simple_seq2seq" / "experiment.json",
+                          self.FIXTURES_ROOT / "data" / "seq2seq_copy.tsv")
+
+    def test_encoder_decoder_can_train_save_and_load(self):
+        self.ensure_model_can_train_save_and_load(
+                self.param_file,
+                overrides="{model: {encoder: {bidirectional: true}}}"
```

Comment:
Why not just change this in `experiment.json`?

Reply:
I'd prefer to test with bidirectional set to both true and false. And, naturally, duplicating the config file wasn't very compelling.

Reply:
Oh, ok.  It looks like you can just add this as another method on the other class, too, instead of making a whole new class for this.